index.Rmd

```diff
@@ -201,12 +201,12 @@ Still Working on it...
 
 # Results
 
-## Top ten most committed crime types
+## Top ten most committed crime types 
```

Comment:
Awesome results! I like it a lot. It would be better if you would use spec_color, spec_font_size or fullwidth to be the more attractive and user-friendly table.